romana-install/roles/romana/kubernetes-postinstall/files/demo/demo.sh

```diff
@@ -21,7 +21,7 @@ function maybe_first_prompt() {
 }
 function run() {
     maybe_first_prompt
-    rate=25
+    rate=$((25 * (1 + ${#1} / 30)))
```

Comment:
Variable multiplier that depends on length of the passed string. It will **type** faster if string is long and slower string is short.